db/migrate/20230117190802_create_workflow_migrations.rb

```diff
@@ -0,0 +1,35 @@
+class CreateWorkflowMigrations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflows do |t|
+      t.references :ems,           :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :tenant,        :type => :bigint, :index => true
+      t.references :git_reference, :type => :bigint, :index => true
```

Comment:
What was this again?  Are we using this? Wondering if we should add git stuff later in a separate set of commits for that feature.

Reply:
Yeah this was to support creating/updating these from a git repo.  I'll drop for now